### PR TITLE
Update compliance-policy-create-windows.md

### PR DESCRIPTION
**Pull request by mariusdo**
Documented per request in https://portal.microsofticm.com/imp/v3/incidents/incident/418222081/summary.

### CODE REVIEW

File: memdocs/intune/protect/compliance-policy-create-windows.md [hunk 1]
```diff
@@ -212,12 +212,14 @@ Applies only to co-managed devices running Windows 10/11. Intune-only devices re
 - **Antivirus**:  
   - **Not configured** (*default*) - Intune doesn't check for any antivirus solutions installed on the device.
   - **Require** - Check compliance using antivirus solutions that are registered with [Windows Security Center](https://blogs.windows.com/windowsexperience/2017/01/23/introducing-windows-defender-security-center/), such as Symantec and Microsoft Defender.
+    Product out of date or with disabled scanning is considered not compliant.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/compliance-policy-create-windows.md:215]:
```suggestion
    A product out of date or with disabled scanning is considered non-compliant.
```

File: memdocs/intune/protect/compliance-policy-create-windows.md [hunk 2]
```diff
@@ -212,12 +212,14 @@ Applies only to co-managed devices running Windows 10/11. Intune-only devices re
...
 
   [DeviceStatus CSP - DeviceStatus/Antivirus/Status](/windows/client-management/mdm/devicestatus-csp)
 
 - **Antispyware**:  
   - **Not configured** (*default*) - Intune doesn't check for any antispyware solutions installed on the device.
   - **Require** - Check compliance using antispyware solutions that are registered with [Windows Security Center](https://blogs.windows.com/windowsexperience/2017/01/23/introducing-windows-defender-security-center/), such as Symantec and Microsoft Defender.
+    Product out of date or with disabled scanning is considered not compliant.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/compliance-policy-create-windows.md:222]:
```suggestion
    A product out of date or with disabled scanning is considered non-compliant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mariusdo, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Brenduns** [comment]
#sign-off

**Pull request merged**